advanced-usage/mocking-non-public-members-and-types.md

```diff
@@ -11,12 +11,26 @@ position: 13
 
 # Mocking Non-public Members and Types
 
-In elevated mode, you can use __Telerik® JustMock__ to mock non-public members and types. That is useful when you want to isolate calls to non-public members.
+This article provides various examples that demonstrate how to mock non-public members and types with Telerik JustMock.
```

Comment:
Could we change the first usage of Telerik JustMock to be with bold style like this: __Telerik® JustMock__. All following mentions of JustMock could be with normal style. This applies to all modified articles.

Reply:
That's done now.